app/src/components/modal/modal_deposit_withdraw/index.tsx

```diff
@@ -92,7 +92,7 @@ export const ModalDepositWithdraw = (props: Props) => {
   const DAI = getToken(1, 'dai')
 
   const wallet = exchangeType === ExchangeType.deposit ? daiBalance : xDaiBalance
-  const minDeposit = exchangeType === ExchangeType.deposit ? parseEther('0.005') : parseEther('10')
+  const minDeposit = exchangeType === ExchangeType.deposit ? parseEther('5') : parseEther('10')
```

Comment:
I would rename this variable since its used both for deposit and withdraw amount. To maybe  minimumExchangeAmount or something similar....